Tests/SentryTests/SentrySpanTests.swift

```diff
@@ -0,0 +1,83 @@
+import XCTest
+
+class SentrySpanTests: XCTestCase {
+    
+    private class Fixture {
+        let someTransaction = "Some Transaction"
+        let someOperation = "Some Operation"
+        let someDescription = "Some Description"
+        let extraKey = "extra_key"
+        let extraValue = "extra_value"
+        
+        func getSut() -> Span {
+            let transaction = Transaction(name: someTransaction)
+            return Span(transaction: transaction, trace: SentryId(), parentId: transaction.spanId)
+        }
+    }
+    
+    private var fixture: Fixture!
+    override func setUp() {
+        fixture = Fixture()
+    }
+    
+    func testInitAndCheckForTimestamps() {
+        let span = fixture.getSut()
+        XCTAssertNotNil(span.startTimestamp)
+        XCTAssertNil(span.timestamp)
+    }
+    
+    func testFinish() {
+        let span = fixture.getSut()
+        span.finish()
+        XCTAssertNotNil(span.timestamp)
+        XCTAssertTrue(span.isFinished)
+    }
+    
+    func testFinishWithStatus() {
+        let span = fixture.getSut()
+        span.finish(status: .ok)
+        XCTAssertNotNil(span.timestamp)
```

Comment:
`m`: Please assert for an actual value. You can use `CurrentDate.setCurrentDateProvider(TestCurrentDateProvider())` to set the date to a fixed date. This again applies to more tests, than just this one.